JumpscaleCore/clients/ssh/SSHClientBase.py

```diff
@@ -429,11 +425,11 @@ def _execute_script(
             content += "\n"
 
         if die:
-            content = "set -ex\n{}".format(content)
+            content = "set -e\n{}".format(content)
 
         dpath = "/tmp/jsxssh_{}.sh".format(j.data.idgenerator.generateRandomInt(1, 100000))
 
-        self.executor.file_write(dpath, content, mode="7770")
+        self.executor.file_write(dpath, content, mode="777")
```

Comment:
to be honest i really believe mode should be int, to be consistent with ssh libs apis and what users expect